CHANGELOG.md

```diff
@@ -4,6 +4,19 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [0.3.0] 2023-05-12
+
+### Added
+Map Clustering for IOS and Android
+
+## [0.2.0] 2022-12-08
+
+### Added
+Source link support
+
+### update
+bump uno to 4.4.20
```

Comment:
I know I'm nit-picky but use some capital letters please. Don't do it if you don't want to.